src/main/java/com/rebuy/consul/RebuyConsulClient.java

```diff
@@ -0,0 +1,36 @@
+package com.rebuy.consul;
+
+import com.ecwid.consul.v1.ConsulClient;
+import com.ecwid.consul.v1.agent.model.NewService;
+
+import java.util.ArrayList;
+
+public class RebuyConsulClient
+{
+    public static void setup(String host, String name, String index, int port)
+    {
+        String id = String.format("%s:%s", name, index);
+
+        NewService.Check check = new NewService.Check();
+        check.setInterval("10s");
+        check.setHttp(String.format(
+            "http://localhost:%d/health",
+            port
+        ));
+
+        NewService service = new NewService();
+        service.setId(id);
+        service.setName(name);
+        service.setPort(port);
+        service.setCheck(check);
+        service.setTags(new ArrayList<>());
+        service.getTags().add("consul-java-client");
```

Comment:
This tag is intended to identify that a certain service is using this library. Not sure if this is useful though ...